src/ResourceManager/ManagementGroups/Commands.ManagementGroups/Cmdlets/RemoveAzureRmManagementGroup.cs

```diff
@@ -0,0 +1,82 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ManagementGroups.Common;
+using Microsoft.Azure.Commands.ManagementGroups.Models;
+using Microsoft.Azure.Management.ManagementGroups;
+using Microsoft.Azure.Management.ManagementGroups.Models;
+
+namespace Microsoft.Azure.Commands.ManagementGroups.Cmdlets
+{
+    /// <summary>
+    /// Remove-AzureRmManagementGroup Cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRmManagementGroup",
+         DefaultParameterSetName = Constants.ParameterSetNames.GroupOperationsParameterSet,
+         SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureRmManagementGroup : AzureManagementGroupsCmdletBase
+    {
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ManagementGroupParameterSet, Mandatory = false, 
+            HelpMessage = Constants.HelpMessages.InputObject, ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public PSManagementGroup ManagementGroup { get; set; }
```

Comment:
@rajshah11 this should be renamed to `InputObject`

Reply:
Done. Thanks.